operator/cmd/mesh/manifest-generate_test.go

```diff
@@ -208,6 +208,38 @@ func TestManifestGenerateGateways(t *testing.T) {
 	checkRoleBindingsReferenceRoles(g, objs)
 }
 
+func TestManifestGenerateIstiodRemote(t *testing.T) {
+	testDataDir = filepath.Join(operatorRootDir, "cmd/mesh/testdata/manifest-generate")
+	g := NewGomegaWithT(t)
+	m, _, err := generateManifest("istiod_remote", "", liveCharts)
+	if err != nil {
+		t.Fatal(err)
+	}
+	objs := parseObjectSetFromManifest(t, m)
+	g.Expect(objs.kind(name.CRDStr).size()).Should(Equal(25))
+	g.Expect(objs.kind(name.ClusterRoleStr).size()).Should(Equal(2))
```

Comment:
Testing size is meaningless in my opinion. We should check the actual resources we expect exists

Reply:
@howardjohn Sorry for the late response.

> We should check the actual resources we expect exists

You mean for all resources or just CRD